Tests/UIKitTests/UIRefreshControlExntesionsTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  UIRefreshControlExntesionsTests.swift
+//  SwifterSwift
+//
+//  Created by ratul sharker on 7/24/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if os(iOS)
+
+import XCTest
+@testable import SwifterSwift
+
+final class UIRefreshControlExtensionTests: XCTestCase {
+
+    func testBeginRefreshAsRefreshControlSubview() {
+        let tableView = UITableView()
+        XCTAssertEqual(tableView.contentOffset, .zero)
+        let refreshControl = UIRefreshControl()
+        tableView.addSubview(refreshControl)
+        refreshControl.beginRefreshing(in: tableView, animated: true)
+
+        XCTAssertTrue(refreshControl.isRefreshing)
+        XCTAssertEqual(tableView.contentOffset.y, -refreshControl.frame.height)
+    }
+
+    func testBeginRefreshAsRefreshControlProperty() {
+        if #available(iOS 10.0, *) {
```

Comment:
@ratulSharker Can you put this test case inside the same method than the other to follow the [Contributing Guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md#adding-tests) one to one relation test method per extension?